templates/customers/verify-code.liquid

```diff
@@ -1,16 +1,19 @@
 {% layout settings.customer_layout %}
-
 <div class="grid">
 
   <div class="grid-item large--one-third push--large--one-third text-center">
+    {% form 'verify_code' %}
+        <input id="provider" name="provider" type="hidden" value="{{ form.provider }}" />
+        <input id="returnUrl" name="returnUrl" type="hidden" value="{{ form.returnUrl }}" />
+        <input id="rememberMe" name="rememberMe" type="hidden" value="{{ form.rememberMe }}" />
+        <input id="username" name="username" type="hidden" value="{{ form.username }}" />
+        <h1>{{ 'verify.code.title' | t }}</h1>
+        {% include 'form-errors-custom' %}
+        <label class="hidden-label" for="code">{{ 'verify.code.label.code' | t }}</label>
+        <input id="code" name="customer[verification_code]" placeholder="{{ 'verify.code.placeholder.enter_code' | t }}" type="text" value="" autofocus/>
+        <label><input id="rememberBrowser" name="customer[remember_browser]" type="checkbox" class="check" value="true"/>{{ 'verify.code.label.checkbox' | t }}</label>
```

Comment:
Checkbox value is always "true". Is it right?

Reply:
Yes. If we do not specify value="true", turned on checkbox would generate "on" value instead of "true" we want for bool in controller model.
If it is unchecked, no value will be passed on submit.

Reply:
No need in this view, templates/verify-code.liquid is used. This one should be removed.